### DISCUSSION TITLE
Laptop Dock

### DISCUSSION THREAD

**Discussion by Irev-Dev** [zalo/CascadeStudio#80] [Show and tell]
It's pretty nice being able plug my laptop into my desktop setup in one movement.

[CascadeStudio link](https://zalo.github.io/CascadeStudio/?code=7RrJbts49O6v0FGqKVmSrcRp0AKTLpNg2mnQpu0h8IGJ6ViIIhmW3AVF%2Fn0eRYp6pCg77oLpoT4kMPn2naRHI%2BcDXaf0KmPlYHBd5GXllBXN569YflMtnSfOYXiM109ZerOs5LrcqOiKrZ%2BzVQ0fK%2FiM0flZ3qxHLfw1Z%2FcxnQv44PAYr18s0%2BvbnJUlxwkmzd4t%2B9pgtOjHmsgYc6wJ%2FZbRtUY3UTLSVVWs9L3g0NhWthgnob6jlJi02i3pmpNqtiKlgtywGGSecjGv2QmrPjOWnxfrqsYMjiTAClYuCq7GcwnKpRkHE23%2Folih7cMgSRQHCgvITUPsnNaPRcbWEjsMohb7ipbKX2081H442SwWbM2liRV4sVik1ynNzrPNDex8Gzjw%2BczxH4PWQZyQemVZh9Jj5yCIxEJW2xlgDoLpgVha03m6KR874yCUWBVNs7dyNQYD3EumC1r%2BDfax8oyCg7HJ8yjRmYJUzmgUHQWJxjgMLHzBtKNYcb4qcpbd9ig7nuqMk2BqanvUsGh5jrs8ISynNdPBaOQ8o1nmrCiEyWKTX1cpyDEYXIDvyoxWzL0MiR%2BFJJkR547eslMReC72i%2BcdGwhgABMB2bQLn3QZtJbg4IM7mubnIKTr1UKf3a1A54pycQeDRvCawLs7UKih8q22HpEmI9IsRBqMIKvce9LeGWuqxyYHEcAPz1MelwyC2a0h%2BOfZ1yzN52ztthSG0VCvOgSVF484CvfSiqxBz2pgr%2F6rC%2FO2qKTVIhKCzY5CgiFsKMjWfuQjNiSsSWjoygblEnIcsHP22Xl3y6rrJaDjgjSKO%2Fr6uC7BvqoCM09pH7xKc3ZR7CYWWpE%2BC2iyfdcXDh%2FFCKqGfJlmoJwr4sBC4acRsMqna%2Bx%2Fl%2Fle5HO3Wm%2BYF7ykEJGe7rD4F3gM6x3v70gU5GEw2YNuiFCt3m63NciGZru2A9nfGzvUcawAfZYFK4TxTo8qbzYJH8ls5QkvNnsAAYxE2wBffAFuc%2BaKBeJcJjVpAwpXjUTytpJD5VFRRCVlhmoKTF6AcFJ8cZsZzFdmITiKjNR4FJNIcBUUkGyikju%2BZTqr03hraPb6yOcGBF5S9qYW9vrCs4J1PKGDaX4Ao8mmZPgCAN%2Fn0NzcSwnXGBnEM%2BCwywSxYdcq2JUCfc2qzToXKzAUaN30FV3fqJ78rWmjf9rnb98%2BbQ3sgX3oB1B%2F1xb2Xb1q%2FKdX%2FelVf3rV79yr9Gb1A6e%2BdOE6bo3mDB3ToZ7zVLuKaZCQQPv1yhr7fmDgf%2FfJVTOEFPRfdgOn40%2BQolJYfC%2B2LjY3S90J1pAMCQ468ARPB9QAjRAdh55n5xU%2FgNmWnNpPjIlFjFNG5%2B9WtL6P2iWJ35Lmadl%2BM1KylbmWazjpSAa1QPtiNVCxUv7Y0yEWjpoVrfzeX0C8nVBemkT3eDG%2FYaW7m7O%2Fi9uB55EJuZySqCkK2mXlP2Y5fHjINRVY5xcFsadVBVl7VG6iACS2xdhcbVxhrDcmM5ZVTLXrrartGq50sahoXKck1LXR7nTBkjX%2BzDse3B%2FzG69OWg%2Fsxa8BKF08kRvYZVuoTdFbBlYNQmK5YOYqddCVBr1CfEyr5TOWV2z9V%2FU6nc8z9mZxUlRVcSdvrVu%2B7fAfmn3SbIyzjjcUQ3QqsBhE07J7Sz7UHwaI%2FnXYuTafkT0V1kK5I%2Fo99rd4Megp8VfsE8vYnKc1aAUdu9hkc9Hm4fuCZiXznCdPUQ%2FrO6%2FEEKP2a7josN5o5k%2BouLax0kmCxD5wOtHYvnVEzHeUHVy4jB2MHfNqozMTQ83zdM2EXZ849RwENGcKMF3oNsTNn3%2B6g5Q4WfrGJNV8bEy5ErwwKMD7gUleH8BMIi0LPCAdyzlj0D2WnvEoLAUyKgIodnABwMu1NWVuExFOxHlNq2UA%2BTMv7lxvFIX6rIf7jBwMpzBDxpOZ50BpmzhxuAs%2BBmNehj3w5qhrmt4yc3J7m2GzC8cyn26tCVsnV9DDzGKtoDfPHaiI86ezcy6KyE%2Be3HUi45RVz2tQGB0fPau1aZRYskhDQ4%2B2lpl8OyH%2FF1HarYrK9K69wFKG7VytHmDAJs%2FUGhE1AU5ZLWUKOuT8ie5VsaiE9XTiHyExIYHMiPPHEBNvFosSZJf%2FenH9OPZI4nmSrWJpkuSHNrHZVUb2eKSLgJx1QY1zIE8htd0PLTNuO7AhsdE4jSzkDdwxiNXOLL4821TyRI7f0m3HRicKifGQ%2Fyg2OUlBG7qalKKEi72uQugWAYeJAG%2FuEEQz5ol8Usy%2FntJsgaKLA9RQ2kUXivGe9y0MgX9E0abE1IJm%2BeHC0KE9BCJbl7VSQNkNgfXDFMKdfZs8LH48i%2F2BHjL%2F3hH5P%2FtKs5OVlK2C%2FgyTew%2B2uXH73xxp1YFRWAJOQEh%2B%2F8BswvgC3nJLgY06hNF%2F0m25CNdGDM8W%2BHzo8KvAiMAZnSTkgBzCIZZEEdbril7fnlN%2BTXhaZEyqFumnODjj26uMidujk%2BXcYRxvugrrtHv4YbX1TaH5lBzt0jjuUTkKkocobd7TYq33UzHuI9%2BvZNyjpfxhF794%2B0Cz%2BmdiMdqolqziP3eqx1taprnrGpnQoHqjcRJ6j6JpOKqBz8%2BUFXkJeqffjuKuQHCNkvmIUbRa1bCr25OCQi7TZ8qGRmfSNJmgjqaWwV72ewVi42I6qeO09rAQBnFXAnP%2BqE1PkIomAjKlEkD2YosVexREFcCCZLtC3yKR5qkkNH3EwdVtr1yFg8B%2F&gui=q1ZKzs8tyM9LzSvxS8xNVbJSSk4sTk5MSQ3LTC1X0lHyTS3OCEotVrIy0DOE84IS89KBSqMN9AwMdYxidZScE5MzUu2VrEqKSlNrAQ%3D%3D)

![thumbnail_main](https://user-images.githubusercontent.com/29681384/109471440-dd4e3180-7ac4-11eb-81fe-10af8806c889.JPG)
![thumbnail_zoom](https://user-images.githubusercontent.com/29681384/109471447-e0e1b880-7ac4-11eb-9d30-df8c14f7557f.JPG)

**Comment by kenweblab** [comment]
Looks great!
